### PR TITLE
Puttering

### PR DESCRIPTION
**Pull request by loganj**

### CODE REVIEW

File: src/main/java/org/coffeetrain/doggifs/ShowGifScreenView.kt [hunk 1]
```diff
@@ -3,33 +3,16 @@ package org.coffeetrain.doggifs
...
-  fun read(inStream: InputStream): ByteArray {
-    val out = ByteArrayOutputStream()
-    val buffer = ByteArray(1024)
-    while (true) {
-      val r = inStream.read(buffer)
-      if (r == -1) break
-      out.write(buffer, 0, r)
+    val gifStream = context.assets.open(flowKey<ShowGifScreen>().path)
+    gifStream.use {
+      val bytes = it.readBytes()
```
Comment (loganj) [inline src/main/java/org/coffeetrain/doggifs/ShowGifScreenView.kt:13]: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-input-stream/read-bytes.html

File: src/main/java/org/coffeetrain/doggifs/MainActivity.kt [hunk 2]
```diff
@@ -5,15 +5,19 @@ import android.content.Context
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.Window
-import android.view.WindowManager
-import flow.*
```
Comment (edenman) [inline src/main/java/org/coffeetrain/doggifs/MainActivity.kt:9]: ooh, where is this setting?  i couldn't find it

File: src/main/java/org/coffeetrain/doggifs/ShowGifScreenView.kt (hunk 1 above)
Comment (edenman) [inline src/main/java/org/coffeetrain/doggifs/ShowGifScreenView.kt:13]: excellent

File: src/main/java/org/coffeetrain/doggifs/MainActivity.kt (hunk 2 above)
Comment (loganj) [inline src/main/java/org/coffeetrain/doggifs/MainActivity.kt:9]: I think I set it crazy high in the Square code style file-- higher than the UI would let me.

**Pull request merged**